crowdin_api/api_resources/translations/resource.py

```diff
@@ -2,6 +2,7 @@
 
 from crowdin_api.api_resources.abstract.resources import BaseResource
 from crowdin_api.api_resources.enums import ExportProjectTranslationFormat
+from crowdin_api.api_resources.translations.types import FallbackLanguages
 from crowdin_api.api_resources.translations.enums import (
     CharTransformation,
     PreTranslationApplyMethod,
```

Comment:
The `PreTranslationApplyMethod` should also be updated - it has the new possible value